src/aggregation.js

```diff
@@ -114,6 +114,7 @@ export default class Aggregation {
     const json = {
       key: this.key,
       initial: this.initial,
+      reduce: this.reduceFn,
       reduceFn: this.reduceFn,
```

Comment:
@thediycoder like we discussed, please add a separate ticket to remove the duplication here `reduce` and `reduceFn`.